tasks/modules/logging/syslog.rb

```diff
@@ -2,9 +2,41 @@
 
 module Logging
   module Syslog
-    def self.log message
+
+    def self.log message, level = nil
+      # Tumbler Collin severity mapping:
+      # - http://www.rubydoc.info/gems/collins_client/0.2.17/Collins/Api/Logging/Severity
+      #
+      # Compared to Syslog, Collins has an additional level 'note', for an operator input.
+      # We should not come into this level via Genesis.
+      # Anyway, handle it via the default level.
+
+      # Default
+      if (level.nil? || level.empty?)
+        severity = ::Syslog::LOG_INFO
+      else
+        severity = case level.downcase.to_sym
```

Comment:
Level.to_s.downcase.to_sym will allow user to pass in a symbol and it not blow up (I'm pretty sure downcase isn't a method on symbols)